scripts/lib-franklin.js

```diff
@@ -469,6 +511,53 @@ export function buildBlock(blockName, content) {
   return blockEl;
 }
 
+/**
+ * Gets the configuration for the given block, and also passes
+ * the config through all custom patching helpers added to the project.
+ *
+ * @param {Element} block The block element
+ * @returns {Object} The block config (blockName, cssPath and jsPath)
+ */
+function getBlockConfig(block) {
```

Comment:
This lets us override the default block loading path via methods pushed to `window.hlx.patchBlockConfig`.
The experimentation plugin expects it to be here to work since it patches the block path for block experiments